paasta_tools/mesos_tools.py

```diff
@@ -115,20 +115,32 @@ def get_current_tasks(job_id):
     return get_mesos_master().tasks(fltr=job_id, active_only=False)
 
 
+def is_task_running(task):
+    return task['state'] == 'TASK_RUNNING'
```

Comment:
I'm going to leave this for now because I don't want to make this change at the same time as everything else.